### PR TITLE
i18n(es): Updated `cli-reference.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `cli-reference.mdx` based on this commit 2ee39246d2396a5e4f4c9dd0e9256180f32214b7

### CODE REVIEW

File: src/content/docs/es/reference/cli-reference.mdx [hunk 1]
```diff
@@ -258,6 +259,25 @@ Agrega una integración a tu configuración. Lee más en la [guía de integracio
 
 Inicia el sitio web de la documentación de Astro directamente desde la terminal.
 
+## `astro info`
+
+Reporta información útil sobre tu entorno Astro actual. Útil para proporcionar información al abrir un issue.
```
Comment (Waxer59) [inline src/content/docs/es/reference/cli-reference.mdx:264]:
```suggestion
Proporciona información útil sobre tu entorno de Astro actual. Útil para proporcionar información al abrir un issue.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**